src/api/helpers/map-CIS-countries/map-CIS-country/can-apply-for-quote-offline/index.ts

```diff
@@ -9,15 +9,15 @@ const { CIS } = EXTERNAL_API_DEFINITIONS;
  * @returns {Boolean}
  */
 export const canApplyForAQuoteOffline = (originalShortTermCover: string) => {
-  if (originalShortTermCover === CIS.SHORT_TERM_COVER_AVAILABLE.ILC) {
+  if (originalShortTermCover === CIS.SHORT_TERM_COVER.ILC) {
```

Comment:
could we dereference up to SHORT_TERM_COVER? not a blocker

Reply:
Updated